terraform/README.md

```diff
@@ -95,25 +119,20 @@ For more information check the Makefile contents.
 
 ## Run bare Terraform commands
 
-You have to use the terraform wrapper script to run terraform commands directly:
+Generally you won't need to run bare Terraform commands as our ```Makefile``` should cover most needed scenarios. However, if you do then you should use the terraform wrapper script to run them directly. The terraform wrapper script takes care of loading your ```.envrc``` and running ```aws-auth```.
 
-```
-cd root/aws-dm
-./../../terraform-wrapper output
-```
-
-This command would have the same effect:
+For example, if you wanted to run ```terraform output``` against preview you can do so:
 
 ```
-cd root/aws-dm
-. .envrc && aws-auth terraform output
+cd environments/preview
+./../../terraform-wrapper output
```

Comment:
This is a bit weird - are we trying to `source` the wrapper - if so, then the command needs to be
```
. ../../terraform-wrapper output
```
and if we're not trying to source it, then the initial `./` is unnecessary.

Reply:
Not trying to source it. I'll drop the initial `./`